db/migrate/20181015173630_add_first_day_and_paid_yet_to_change_report.rb

```diff
@@ -0,0 +1,6 @@
+class AddFirstDayAndPaidYetToChangeReport < ActiveRecord::Migration[5.2]
+  def change
+    add_column :change_reports, :first_day, :datetime
+    add_column :change_reports, :paid_yet, :integer
```

Comment:
After the column for `paid_yet` is added, the default value should be set to 0.

`change_column_default :change_reports, :paid_yet, from: nil, to: 0`